### DISCUSSION TITLE
External merge and diff tooling support

### DISCUSSION THREAD

**Discussion by Wickkawizz** [eclipse-jgit/jgit#62] [General]
Hi everyone, I'm looking to use JGit for a school project and was wondering if there was any ETA for supporting external merge and diff tools. I will be integrating Egit/Jgit with a custom 3-way differencing tool that has been built on Eclipse and found that EGit would be the best tool, since it already has a lot of the Eclipse tooling integration done for me.

**Comment by iloveeclipse** [comment]
It is already there since 6.2: 
- https://wiki.eclipse.org/EGit/New_and_Noteworthy/6.2#:~:text=To%20configure%20an%20external%20diff,tool%20from%20the%20combo%20box.
- https://wiki.eclipse.org/JGit/New_and_Noteworthy/6.2

**Reply by Wickkawizz** [reply]
I based my question on the github page where the documentation said it was an unimplemented feature. Didn't know it had a full wiki with the information there. Thanks for the references!
